gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Jun 01 08:01:38 UTC 2021
+#Mon Feb 21 14:38:17 CET 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
```

Comment:
When you update gradle, you need to call the gradlew. 
`./gradlew wrapper --gradle-version=7.2`

Reply:
Done, thanks!